CHANGES

```diff
@@ -1,10 +1,16 @@
 Release 1.4.5 (in development)
 ==============================
 
+Features added
+--------------
+
+* latex writer allows user customization of topic "shadow" boxes, via three lengths variables
+
 Bugs fixed
 ----------
 
 * #2676: (latex) Error with verbatim text in captions since Sphinx 1.4.4
+* new config option ``latex_macros_onlywithprefix`` to help resolve macro name conflicts with latex packages or classes such as memoir (ref #2629)
```

Comment:
This is surely a bug fix, but it is also new feature. So the entry should move or copy to *Feature added* section to let users know about the new option.